test/orm/declarative/test_tm_future_annotations.py

```diff
@@ -293,11 +290,11 @@ class MyClass(decl_base):
             data: Mapped[str] = mapped_column(default="some default")
 
             if reference_type.container_w_remote_mapped:
-                status: ClassVar[Dict[str, MyOtherClass]]
```

Comment:
this is testing`Dict[]` which we need to keep testing.   This file would overall need more tests to ensure both forms work